de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/filesystem/IResource.java

```diff
@@ -50,4 +50,8 @@
   public IPath getLocation();
 
   public Object getAdapter(Class<? extends IResource> clazz);
+
+  /** Returns the {@link IReferencePoint} on which the resource is referenced to */
+  @Deprecated
```

Comment:
is it deprecated because it is under development and you dont want other users to use it?

Reply:
its because in the final state, a saros resource should'nt know its reference point. But during the refactoring, But during refactoring, it is confortable that a resource knows its reference point